lib/google-auth.js

```diff
@@ -0,0 +1,41 @@
+import google from 'googleapis';
+import util from './util';
+
+class GoogleAuth{
+
+    authenticate(callback) {
+        if (this.jwtClient) {
+            this.jwtClient.authorize((err, tokens) => {
+                if (err) {
+                    console.error('An error occured when authenticating to Google API.');
+                    console.error(err);
+                    return;
+                }
+                callback(tokens);
+            });
+        } else {
+            let apiKeyFileName;
+            if (process.env.NODE_ENV === 'TEST') {// FIXME This is ugly. We shouldn't have to change code in order to run tests...
+                apiKeyFileName = 'google-service-api-json-key.example.json';
+            } else {
+                apiKeyFileName = 'google-service-api-json-key.json';
+            }
+            util.readJsonFromFile(apiKeyFileName , (err, data) => {
+                if (err) {
+                    console.error('An error occured when loading the Google Api Key File.');
+                    console.error(err);
+                    return;
+                }
+                this.jwtClient = new google.auth.JWT(
+                    data.client_email,
+                    null,
+                    data.private_key,
+                    ['https://www.googleapis.com/auth/drive.metadata.readonly', 'https://spreadsheets.google.com/feeds'],
```

Comment:
How about we extract these URLs as `const`s int he top of the file? I think it will read better and better communicate their purpose in this JWT method here... I think it also makes it easy to change them in the future...